plugins/modules/cloud/google/gcp_storage_aws_transfer_job.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Terry Jones <terry.jones@example.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: gcp_storage_aws_transfer_job
+version_added: 1.0.0
+short_description: Creates GCP Transfer Jobs between AWS S3 and GCP Storage.
+description: This module will create GCP Storage Transfer Jobs which transfers AWS S3 Bucket objects into the specified
+             GCP Bucket. If there is an existing Transfer Job with the same Description it will delete it and create a new job.
+             If you include the parameter aws_s3_bucket_prefix, it will transfer that prefix only, otherwise it will transfer all
+             objects in the bucket. The Transfer Jobs will only pull changed or new objects when they execute.
```

Comment:
```suggestion
description:
    - This module will create GCP Storage Transfer Jobs which transfers AWS S3 Bucket objects into the specified GCP Bucket.
     - If there is an existing Transfer Job with the same description it will delete it and create a new job.
     - If you include the parameter I(aws_s3_bucket_prefix), it will transfer that prefix only, otherwise it will transfer all objects in the bucket.
     - The Transfer Jobs will only pull changed or new objects when they execute.
```
IMO, easier to read.